src/SDKs/Search/DataPlane/Search.Tests/Tests/AutocompleteTests.cs

```diff
@@ -156,13 +156,14 @@ protected void TestAutocompleteOneTermWithContextWithFuzzy()
 
         protected void TestAutocompleteCanUseHitHighlighting()
         {
-            var expectedText = new List<String>() { "police", "polite", "pool", "popular" };
-            var expectedQueryPlusText = new List<String>() { "<b>police</b>", "<b>polite</b>", "<b>pool</b>", "<b>popular</b>" };
+            var expectedText = new List<String>() { "police", "polite" };
+            var expectedQueryPlusText = new List<String>() { "<b>police</b>", "<b>polite</b>" };
 
             SearchIndexClient client = GetClientForQuery();
             var autocompleteParameters = new AutocompleteParameters()
             {
                 AutocompleteMode = AutocompleteMode.OneTerm,
+                Filter = "baseRate gt 200 or baseRate lt 100",
```

Comment:
@ishansrivastava90 There is no top-level `baseRate` property anymore after my Complex Types changes (I'm assuming this is using the test `Hotel` type). Please pick a different property on which to filter. Sorry for the inconvenience.